src/components/sbb-tab-group/sbb-tab-group.tsx

```diff
@@ -101,14 +101,16 @@ export class SbbTabGroup implements ComponentInterface {
   }
 
   private get _enabledTabs(): InterfaceSbbTabGroupTab[] {
-    return this.tabs.filter((t) => !isValidAttribute(t, 'disabled'));
+    return this.tabs.filter(
+      (t) => !isValidAttribute(t, 'disabled') && interactivityChecker.isVisible(t),
+    );
   }
 
   public connectedCallback(): void {
     this._isNested = !!hostContext('sbb-tab-group', this._element);
   }
 
-  public componentWillLoad(): void {
+  public componentDidLoad(): void {
```

Comment:
Why this change is needed?

Reply:
To be able to use the `getComputedStyle` and make the interactivity check :)